releasenotes/notes/replace-schedule-component-base-1459bf920aa8b784.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes the unecessary specificity of the ScheduleComponent abstract base class for Schedule and Instruction by replacing it with NamedValue. Refer to `#5076 <https://github.com/Qiskit/qiskit-terra/issues/5076>` for more details 
```

This should be a deprecation
```suggestion
    The ``ScheduleComponent`` has been replaced by :class:`qiskit.pulse.values.Value` and :class:`qiskit.pulse.values.NamedValue``. Anywhere that currently accpts a ``ScheduleComponent`` should instead accept a ``Union[Schedule, Instruction]``.
```

---

Will mark it as such